app/views/cp_mailer/nag_email.text.erb

```diff
@@ -3,7 +3,7 @@ Dear <%= @contract[:applicant][:first_name] %> <%= @contract[:applicant][:last_n
 
 You have yet to accept or reject your offer for the TAship of <%= @contract[:position] %>.
 
-If you have not accept your offer by <%= @contract[:deadline] %>, your offer will be resceded. Check your dates and decide soon. This is your <%= @contract[:nag_count] %><%= @contract[:nag_suffix] %> warning.
+If you have not accept your offer by <%= @contract[:deadline] %>, your offer will be revoked. Until you have accepted the offer, we cannot pay you. If you plan to reject the offer, please do so promptly. If you have any questions, please contact the course instructor or the TA coordinator at tacoord@cs.toronto.edu. This is your <%= @contract[:nag_count] %><%= @contract[:nag_suffix] %> warning.
```

Comment:
Please move the first sentence to be the second last sentence.  E.g.

Until you have ... promptly. If you have any... .edu. If you have not accepted your offer ...  revoked.  This is your...

Also, change "accept" to "accepted".